### DISCUSSION TITLE
I lost the access suddenly

### DISCUSSION THREAD

**Discussion by Salahw7** [community/community#14353] [Copilot Conversations] [answered]
I just got access for Co-Pilots but I just lost it...

I tried to connect my second pc with my git account and try to install it and I lost it. Suddenly my account is no longer recognized and shows me unauthorized users.

Thank you for helping me.

**Comment by Salahw7** [comment]
![image](https://user-images.githubusercontent.com/89655562/161883010-1b4db381-4c90-4e20-bbe6-ee3bb1b839b9.png)

**Comment by Salahw7** [comment]
I received the email that confirms that I am taken in the testers, I was also able to do some work on it It was excellent! But I just lost access, because I wanted to connect my laptop to git and install Co-pilots to have access there too and I received the unauthorized error message like when I didn't have access.

**Reply by Salahw7** [reply]
![image](https://user-images.githubusercontent.com/89655562/161893572-a69208aa-ea1c-41f5-859d-21de11971497.png)

**Comment by davecheney** [comment, marked as answer]
Please try again, it should be fixed.

**Reply by Salahw7** [reply]
thank you!
